client/rest-exchangers.go

```diff
@@ -122,3 +124,45 @@ func (restClient *RestClient) CreateExchangerTransaction() gin.HandlerFunc {
 		}
 	}
 }
+
+func (restClient *RestClient) GetExchangerTransactionMinimumAmount() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		type RequestGetTransactionMinAmount struct {
+			From string `json:"from"`
+			To   string `json:"to"`
+		}
+
+		var requestData RequestGetTransactionMinAmount
+		err := decodeBody(c, &requestData)
+		if err != nil {
+			restClient.log.Errorf("GetExchangerTransactionMinimumAmount: Failed to decode body: %s\t[addr=%s]",
+				err.Error(), c.Request.RemoteAddr)
+			c.JSON(http.StatusBadRequest, gin.H{"code": http.StatusBadRequest, "message": msgErrRequestBodyError})
+			return
+		}
+
+		changellyExchanger, _ := restClient.
```

Comment:
Error handling is missing, please add a function that would report an error to the clients, function prototype:
```
func reportError(c *gin.Context, httpErrorCode int, err error)
```